tools/cldr-apps/js/src/esm/cldrForumPanel.js

```diff
@@ -311,6 +77,32 @@ function getUsersValue(theRow) {
   return null;
 }
 
+function havePosts(nrPosts, forumDivClone, tr, loader2) {
```

Comment:
this function was nested, now top-level with 3 additional parameters, otherwise unchanged